Fitz/Features/AccountsRework/Account.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Fitz.Features.AccountsRework
+{
+    public class Account
```

Comment:
I'm thinking this file will evolve and it's not perfect. It's pretty much identical to CreateAccountModel. That wasn't my intention. I'm thinking this model will contain more information later. Maybe its not needed.